zerver/lib/test_helpers.py

```diff
@@ -205,7 +205,7 @@ def __init__(self, post_data, user_profile, assert_callback=None):
         self._log_data = {} # type: Dict[str, Any]
         self.META = {'PATH_INFO': 'test'}
 
-class AuthedTestCase(TestCase):
+class   AuthedTestCase(TestCase):
```

Comment:
looks like an accidental hunk in your commit.  @TomaszKolek I recommend doing a quick self-review of your commit before (or just after) pushing them to a PR to catch little things like this.